.rubocop.yml

```diff
@@ -101,6 +101,7 @@ Metrics/BlockLength:
     - '*/dependabot-*.gemspec'
 
 Metrics/ParameterLists:
+  Max: 6
```

Comment:
I'm not sure that we want to globally increase this, could we add inline exceptions for the methods that need it? Globally relaxing such rules is a bit of a slippery slope IMO, and we're already not in fantastic shape regarding linting in this project.

Reply:
I do agree. I think this could be solved with the `provider_metadata` suggestion from @feelepxyz below.